kubernetes/assets/recommended_monitors/monitor_high_disk_usage_node.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Node High Disk Usage",
+	"type": "query alert",
+	"query": "avg(last_10m):avg:system.disk.in_use{*} by {kubernetes_cluster,host} * 100 > 88",
```

Comment:
avg space aggregator used + hosts can have multiple partitions. in this case, maybe have by device in the group by clause.

Reply:
Check the behaviour - will it trigger for hosts that are not on k8?

Reply:
check if the time aggregator is what we want in this case (suggestion: use min or max, or min over last 5 mins)

Reply:
this isn't specific to kubernetes